schema/handler.go

```diff
@@ -66,6 +102,7 @@ func (h *Handler) get(w http.ResponseWriter, r *http.Request, ps httprouter.Para
 		src = resp.Body
 	}
 
+	w.Header().Add("Content-Type", "application/json")
```

Comment:
/cc @zepatrik just a quick reminder that we need to set the content type explicitly when not using herodot :)